prac_06/guitar_test.py

```diff
@@ -0,0 +1,10 @@
+from guitar import Guitar
+
+
+gibson = Guitar("Gibson L-5 CES", 1922, 16035.40)
+another = Guitar("Another Guitar", 2013, 500)
+
+print(f"{gibson.name} get_age() - Expected 100.")   # got 100
+print(f"{another.name} get_age() - Expected 9.")   # got 9
+print(f"{gibson.name} is_vintage() - Expected True")    # got True
+print(f"{another.name} is_vintage() - Expected False")  # got False
```

Comment:
I think you should use def function

Reply:
The Code is well-designed. And there is no issue